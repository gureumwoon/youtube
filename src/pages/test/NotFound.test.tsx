import { render } from "@testing-library/react"
import { withRouter } from "../../test/utils"
import { Route } from "react-router-dom"
import NotFound from "../NotFound"

describe('NotFound', () => {
    it('renders correctly', () => {
        const { asFragment } = render(withRouter(
            <Route path="/" element={<NotFound />} />
        ));

        expect(asFragment()).toMatchSnapshot();
    })
})