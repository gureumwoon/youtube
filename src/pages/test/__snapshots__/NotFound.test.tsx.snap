// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<DocumentFragment>
  <div>
    NotFound
  </div>
</DocumentFragment>
`;
